omod/src/main/webapp/resources/scripts/diagnoses/diagnoses.js

```diff
@@ -1,10 +1,14 @@
 (function(diagnoses, $, _, undefined) {
 
-    var codingSystemToUse = 'ICD-10-WHO';
-
     var mapTypeOrder = [ "SAME-AS", "NARROWER-THAN" ]
+    
+    findPreferredCodingSource = function() {
+        var preferredCodingSource = $("#diagnosis-search").attr("preferredcodingsource");
+        return preferredCodingSource;
```

Comment:
What happened to the default preferred coding source?

I was asking to refactor this method so that it would return the default coding source (ICD-10-WHO) if nothing was specified as a attribute, instead of handling the default in the findConceptMapping method, not remove it entirely... or is this handled somewhere else now?  See my pseudo-code in the previous review.

Reply:
Thank you @mogoodrich, yes this is handled at the back end in the Element here https://github.com/openmrs/openmrs-module-coreapps/pull/196/files#diff-7d6c8132f77707ede46ad8ec6a618a59R80 . @mks-d highlighted on this on the previous comment.

Reply:
Cool, thanks!